lib/tasks/seed_app.rake

```diff
@@ -1,8 +1,8 @@
 desc "Create a new seed app including a random auth_secret for local development"
 task :seed_app, [:app_name] do |_, opts|
-  seed_app_filename = Rails.root.join('db/seed_apps.yml')
+  seed_app_filename = Rails.root.join('db', 'seed_apps.yml')
```

Comment:
This is my least favorite cop behavior. especially given this: https://twitter.com/tenderlove/status/842064491936280576

The description of this cop says it's doing it to allow for platforms on which separator isn't `/`, which of course according to the above tweet isn't true of any platform in this universe. So it is basically just making your code more verbose.

The path join methods are a nice convenience (I love using them), and ensuring that no escaping magic is used to fool you when concatenating in user input is maybe good (though I'm not sure it'd be sufficient to prevent even `../..`-style attacks), but forcing you not to use slashes in your literal string paths is just for show.

What do you think?

Reply:
i feel like we should just disable this cop.

Reply:
fine with disabling, although its enabled in rubocop-betterment